docs/source/merlin_server.rst

```diff
@@ -0,0 +1,70 @@
+Merlin Server
+=============
+The merlin server command allows users easy access to containerized broker 
+and results servers for merlin workflows. The main configuration will be
+stored in the subdirectory called "server/" by default in the main merlin
+configuration "~/.merlin". However different server images can be created
+for different use cases or studies just by simply created a new directory
+to store local configuration files for merlin server instances.
+
+Below is an example of how merlin server can be utilized.
+
+First create and navigate into a directory to store your local merlin 
+configuration for a specific use case or study.
+
+.. code-block:: bash
+
+   mkdir study1/
+   cd study1/
+
+After wards you can instantiate merlin server in this directory by running
+
+.. code-block:: bash
+
+   merlin server init
+
+A main server configuration will be created in the ~/.merlin/server and a local
+configuration will be created in a subdirectory called "merlin_server/"
+
+We should expect the following files in each directory
+
+.. code-block:: bash
+
+   ~/study1$ ls ~/.merlin/server/
+   docker.yaml  merlin_server.yaml  podman.yaml  singularity.yaml
+
+   ~/study1$ ls
+   merlin_server
+
+   ~/study1$ ls merlin_server/
+   redis.conf  redis_latest.sif
+
+The main configuration in "~/.merlin/server" deals with defaults and 
+technical commands that might be used for setting up the merlin server
+local configuration and its containers. Each container has their own
+configuration file to allow users to be able to switch between different
+containerized services freely.
+
+The local configuration "merlin_server" contains configuration specific to
```

Comment:
I think this needs to be rewritten. Perhaps

The local configuration "merlin_server" **folder** contains configuration **files** specific to a certain use case or run.

Reply:
changes made